lib/Yam/Agama/Pom/RebootPage.pm

```diff
@@ -23,6 +23,7 @@ sub new {
 sub expect_is_shown {
     my $self = shift;
     my $timeout = 2400;
+    $timeout *= 2 if get_var("MACHINE") eq "ppc64le-emu";
 
```

Comment:
Emulated ppc64 is very experimental, I woudn't expect it to live longer than next week

Reply:
it is quite painful for what I can see, timeouts and slow all over the place and hacks, but for now we can know what happens in ppc64 in agama while we find the time to setup powervm: https://progress.opensuse.org/issues/169480.
It will be great to have power 10 with kvm to scale better with this, from the point of view of the automation it is more straightforward.

Reply:
@rfan1 @tinawang123 - can you help/guide @jknphy with this as part of #167893 (we will need it for SLE 16 anyway)

Reply:
I added my manual test steps to https://progress.opensuse.org/issues/169480. please feel free to ping me if you need access the setup to do some verificatioin.